src/SFA.DAS.EmployerRequestApprenticeTraining.Web/Views/EmployerRequest/SubmitConfirmation.cshtml

```diff
@@ -0,0 +1,83 @@
+﻿@using SFA.DAS.EmployerRequestApprenticeTraining.Web.Extensions
+@using SFA.DAS.EmployerRequestApprenticeTraining.Infrastructure.Configuration
+@using SFA.DAS.EmployerRequestApprenticeTraining.Domain.Types
+@using SFA.DAS.EmployerRequestApprenticeTraining.Web.Controllers
+@model SFA.DAS.EmployerRequestApprenticeTraining.Web.Models.EmployerRequest.SubmitConfirmationEmployerRequestViewModel
+
+@{
+    ViewData["Title"] = "We've shared this with training providers";
+    ViewBag.GaData.Vpv = "/accounts/employerrequests/submitconfirmation";
+}
+<main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content">
+    <partial name="_ValidationSummary" />
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <div class="govuk-panel govuk-panel--confirmation">
+                <h1 class="govuk-panel__title">
+                    @ViewData["Title"]
+                </h1>
+            </div>
+            <p class="govuk-body">We've sent a confirmation email to <strong>@Model.RequestedByEmail</strong></p>
+            <h2 class="govuk-heading-m">
+                Request for @Model.StandardTitle (level @Model.StandardLevel)
+            </h2>
+            <dl class="govuk-summary-list govuk-!-margin-bottom-9">
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        How many apprentices
+                    </dt>
+                    <dd class="govuk-summary-list__value">
+                        @Model.NumberOfApprentices
+                    </dd>
+                </div>
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        Apprenticeship locations
+                    </dt>
+                    <dd class="govuk-summary-list__value">
+                        @Model.SingleLocation
+                    </dd>
+                </div>
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        Training options
+                    </dt>
+                    <dd class="govuk-summary-list__value">
+                        @if (Model.GetTrainingOptions().Count == 1)
```

Comment:
Maybe we could use a HTmlHelper extension to encapsulate this, then it could be reused on other screens if we're using the same logiv to show training options?